aospy_synthetic/db/sqlalchemy/sqlalchemy_config.py

```diff
@@ -89,23 +92,29 @@ def __init__(self, session, AospyObj):
                     setattr(self, key,
                             self._get_db_obj(self._db_attrs[key]['db_cls'],
                                              session,
-                                             getattr(AospyObj,
-                                                     self._db_attrs[key]['obj'])))
+                                             sub_obj))
+
+    def __repr__(self):
+        repr = ''
+        for attr in self.__class__._metadata_attrs:
+            repr += '{}: {}\n'.format(attr, getattr(self, attr))
+            return repr
```

Comment:
This `return repr` should be after the for loop right?

We should ultimately have tests for our `repr`s...I think I remember seeing that xarray has some for theirs.